Documentation/subcommands/fetch.md

```diff
@@ -7,7 +7,7 @@ If the ACI exists locally, it won't be re-downloaded.
 
 The easiest way to fetch an ACI is through meta discovery.
 rkt will find and download the ACI and signature from a location that the creator has published on their website.
-This process is detailed in the [Application Container specification][appc-discovery].
+This process is detailed in the [Application Container specification](https://github.com/appc/spec/blob/master/spec/discovery.md) appc-discovery.
```

Comment:
Please rewrite as follows, an let's make these [reference-style links](https://github.com/coreos/docs/blob/master/STYLE.md#marking-down-the-link) while we're at it: e.g., 
```
The [ACI discovery mechanism is detailed in the App Container specification][appc-discovery].

[...]

[appc-discovery]: https://github.com/appc/spec/blob/master/spec/discovery.md
EOF
```